### PR TITLE
Expand CONTRIBUTING.md to cover Slack + community meetings

### PR DESCRIPTION
**Pull request by webchick**
Doing my homework for https://github.com/ossf/DevRel-community/issues/35 required a bit of spelunking because there wasn't mention of either the Slack channel or the monthly community meetings in either README.md or CONTRIBUTING.md (first two places I looked).

This is a simple PR to just add a reference to those in, in order to help the next person.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ Contributions to gittuf can be of several types:
   enhancements to the implementation
 * new issues or feature requests
 
+You can also join our community in #gittuf in [OpenSSF Slack](https://openssf.slack.com/) and/or our monthly gittuf Community Meetings, which take place the first Friday of the month at 17:00 UTC ([your timezone](https://time.is/1700_UTC)). Here are the prior [meeting notes](https://docs.google.com/document/d/1tXFCVUHsICLpLKxcGvhzBDUWmpsY1LQvysFaX6AJRkk/edit#heading=h.9m0zi4b0wnne) to get oriented.
```
Comment (adityasaky) [inline CONTRIBUTING.md:9]: What do you think about pointing to https://github.com/gittuf/community/blob/main/README.md#join-us so we have this information in a single place?

Comment (adityasaky) [reply CONTRIBUTING.md:9]: nit: Also, could you please wrap the final text to 80 cols? :)

**Review by adityasaky — COMMENTED** [review]
Thanks for opening this! I have a quick comment about this change.

File: CONTRIBUTING.md (hunk 1 above)
Comment (webchick) [reply CONTRIBUTING.md:9]: That is indeed a _much_ better plan (sorry, I missed that!), so yes, will do! And no worries on 80 char wrap, I wasn't sure. :D

Comment (adityasaky) [reply CONTRIBUTING.md:9]: Thanks! Also, could you DCO sign off to make the bot happy?

**Comment by webchick** [comment]
Ok, I _think_ this is now updated in a smarter way and the DCO bot is happy. :)

**Review by adityasaky — APPROVED** [review]
Thanks! I took the liberty of fixing a conflict by pushing to your branch, I hope that's okay. :)

**Comment by webchick** [comment]
Rock! TYSM! 😄

**Pull request merged**